src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx

```diff
@@ -90,6 +90,17 @@ When you edit the site, include the Jira issue key (DOC-1234, for example) in yo
       </td>
     </tr>
 
+        <tr>
+      <td>
+        User story
+      </td>
+
+      <td>
+        *According to [Atlassian](https://www.atlassian.com/agile/project-management/user-stories) "A user story is an informal, general explanation of a software feature written from the perspective of the end user. Its purpose is to articulate how a software feature will provide value to the customer. 
```

Comment:
If you look at your Gatsby preview https://docswebsitedevelop-sdagileuserstories.gtsb.io/docs/agile-handbook/appendices/ticket-best-practices/#jira_checklist , you will see that this is not rendering quite right.